README.md

```diff
@@ -140,19 +140,18 @@ aws ec2 run-instances \
     --key-name MYKEYHERE \
     --security-group-ids sg-7e8b830f
 ```
-##### Create EC2 with Automatic SSL Certification
-The AWS-CLI command below will automatically create an EC2 server and create an SSL certificate from Lets Encrypt, for free.
-```bash
-wget https://raw.githubusercontent.com/statping/statping/master/servers/ec2-ssl.sh
-# Edit ec2-ssl.sh and insert your domain you want to use, then run command below.
-# Use the Security Group ID that you used above for --security-group-ids
 
-aws ec2 run-instances \
-    --user-data file://ec2-ssl.sh \
-    --image-id ami-7be8a103 \
-    --count 1 --instance-type t2.nano \
-    --key-name MYKEYHERE \
-    --security-group-ids sg-7e8b830f
+##### Add HTTPS To A Current Install
+The below steps will convert an EC2 install without SSL to one with working SSL
+```bash
+    -add server.crt and server.key to /statping
+    -stop container docker stop <name>
+    -stop docker service systemctl stop docker
+    -go to directory /var/lib/docker/containers/hash of container
+    -edit file hostconfig.json, change port 8080 and 80 to 443
+    -edit file config.v2.json, change port 8080 to 443
+    -start docker service systemctl start docker
```

Comment:
Please separate commands from comments by using `# `.

Since you are declaring this as `bash` syntax please use the `bash` syntax.

Reply:
Thank you.  I will update and resubmit.  I am a first time user of statping and it was difficult to get started when the instructions are out of date.  I am trying to resolve this for future users and the instructions from Vinay worked on my install.

Reply:
Yea we know that the instructions are out of date.
Thanks for your effort.

Reply:
But first of all:
Those steps are _not_ recommended.
You are editing the raw Docker data structures which shouldn't be done!

I think the complete AWS setup is broke and needs to be rewritten (and documented).

<details>

<summary>WIP/ Unfinished</summary>
The recommended way to enable SSL (via LetsEncrypt) on an AWS EC2 instance is:
(:warning: LetsEncrypt does _only_ issue certificates for _Domain names_ and not for IP addresses! :warning:)

```
## create the required Security Groups
aws ec2 create-security-group --group-name StatpingPublicHTTP \
     --description "Statping HTTP Server on port 80 and 443"
# will response back a Group ID. Copy ID and use it for --group-id below.

## allow traffic on Port 80
aws ec2 authorize-security-group-ingress \
     --group-id sg-7e8b830f --protocol tcp \
     --port 80 --cidr 0.0.0.0/0

## allow traffic on Port 443
aws ec2 authorize-security-group-ingress \
     --group-id sg-7e8b830f --protocol tcp \
     --port 443 --cidr 0.0.0.0/0




```
</details>

Reply:
@MMeffert I've updated the Wiki on multiple pages earlier today, let me know what you found to be difficult to understand and I'll edit it. 💃

Reply:
I found the SSL EC2 part confusing on the README page.  I tried downloading the "ec2-ssl.sh" but it seems to no longer exist.  I followed the instructions on the EC2 part of the wiki and it was easy to follow and worked perfectly.  Maybe update the README page to be more of an index and direct users to the wiki.  That would keep consistent information and reduce the number of places that the information needs to be updated.  I ended up going with the t3.nano instead of the suggestion t2.nano.  It is the newer technology and slightly cheaper.  I have only been running to for a few days but it appears to be running fine.